packages/generators/add-generator.ts

```diff
@@ -94,7 +93,8 @@ export default class AddGenerator extends Generator {
 		config: {
 			configName?: string,
 			topScope?: string[],
-			item?: string;
+			item?: string,
+			merge?: object,
```

Comment:
`object` is way too generic. Is there a proper type for this `merge` option?

Reply:
Added corresponding type @ematipico :+1: